src/providers/ibmcloud/mod.rs

```diff
@@ -159,6 +202,11 @@ impl Drop for IBMGen2Provider {
     }
 }
 
+#[derive(Debug, Deserialize, Clone)]
+struct VendorDataCloudConfig {
```

Comment:
Nit: a docstring on this one would be appreciated.

Reply:
I added a docstring in the next commit.